consent_manager/js_api/commands.rst

```diff
@@ -115,10 +124,8 @@ Code::
 
             {consents: {analytics: {status: 1}}}
 
-    Where ``consent.analytics`` is consent type and status indicate:
```

Comment:
This was removed for `Set compliance settings` but stayed for `Get compliance settings`. IMO it should stay for both as the available status values are different.